### ISSUE TITLE
Feature: rebase branches only marked custom labels

### ISSUE THREAD

**Issue by splincode** [peter-evans/rebase#102]
The main reason is that we only want to autorebase branches that have been approved. Then there will be no big load on the github actions when we have 100500 branches.

Linked: https://github.com/peter-evans/rebase/issues/101

Expected behavior:

```
name: Rebase onto main

on:
    push:
        branches: [main]
    pull_request_review: # support when trigger approved
        types: [submitted]

jobs:
    rebase:
        runs-on: ubuntu-latest
        steps:
            - name: Autorebase
              uses: peter-evans/rebase@v1.2.0
              with:
                  base: main
                  exclude-drafts: true # skip drafted which approved
                  token: ${{ secrets.TINKOFF_BOT_PAT }}
                  exclude-labels: |
                      no-rebase # also skip rebase
                      dependencies # also skip rebase

                  included-labels: |
                      Ready to Merge # rebase branches only ready to merge

                  included-count-approvers: 2 # or instead if you want check also count approve on branch
```

@peter-evans what do you think?

**Comment by peter-evans** [comment]
@splincode I've added the `include-labels` input to the action. It's released as `v1.3.0` / `v1`.

As for the `included-count-approvers` idea. I think that should be something handled on your side. For example, you could automate adding a label based on that condition.

**Comment by splincode** [comment]
Thank you very much!
